spec/vcr_setup.rb

```diff
@@ -0,0 +1,34 @@
+require 'vcr'
+require 'active_support/core_ext/string'
+
+def filter_body_field(c, field)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)